server/message.go

```diff
@@ -22,7 +22,7 @@ func NewMessage(from *db.User, destIp string, text string) *Message {
 	return &Message{
 		SourceMAC: (*from).MAC, SourceIP: (*from).IP,
 		DestIP: destIp, Text: text,
-		StartProto: ""}
+		Handshake: false, StartProto: ""}
```

Comment:
@wavyllama Same thing - let's leave it even though we know it's changed. We'll resolve the merge conflicts in the other branch as they come up.